examples/compose/vtorc-up.sh

```diff
@@ -36,6 +39,14 @@ else
     cp /vt/orchestrator/default.json /vt/orchestrator/config.json
 fi
 
+sleep $sleeptime
+
+# set the correct durability policy for the keyspace
+for keyspace in $keyspaces; do
+  echo "Set Keyspace Durability Policy ${keyspace} to none"
+  /vt/bin/vtctldclient --server vtctld:$GRPC_PORT SetKeyspaceDurabilityPolicy --durability-policy=none $keyspace
+done
```

Comment:
@GuptaManan100 It makes sense!

I put this change here because:

- Once vtorc skips the vttablets, setting a durability policy won't trigger vtorc to run PlannedReparentShard toward these vttablets. I learned from my experiment that either executing PlannedReparentShard manually or restarting vtorc after setting a policy is required after all. I think setting a policy before running vtorc is the safest way to prevent vtorc from skipping the vttablets.
- This change is the least amount of diff among what I came up with.

Anyway, do you have any suggestions in mind?
As the explicit approach, how about adding a new one-off container (like schemaload_lookup_keyspace) to set a policy after (depends_on) all the vttablets?

Reply:
@yoheimuta No No, your understanding of VTOrc is incorrect, it will pickup the durability policy even after its startup. It will just take some time because the keyspace and shard informations are reloaded every 15 seconds, so vtorc will run PRS, but after 15 seconds. 
I understand that is not ideal for an example setup, nobody wants to wait that long for the entire cluster to be ready.

The explicit approach sounds cleaner to me. We never know what other steps of initialisation we might add in the future and adding it to the VTOrc up doesn't seem like the best solution because it makes it imperative for VTOrc to be part of the test setup, even though some people might want to skip it.
We can add the new container with dependencies on the vttablets and VTOrc have a dependency on that container. Then, when VTOrc comes up, the durability policy would already be set.
Does that make sense to you?

Reply:
> it will pickup the durability policy even after its startup.

@GuptaManan100 You're right. I don't have a doubt in this point.

>> Once vtorc skips the vttablets

My point is `it won't pickup the durability policy after its skip decision.`

>> It will just take some time because the keyspace and shard informations are reloaded every 15 seconds, so vtorc will run PRS, but after 15 seconds.

You mean `it will pickup the durability policy even after its skip because vtorc will evetually run PRS anytime ~ 15 seconds after executing SetKeyspaceDurabilityPolicy?` Reloading **every** 15 seconds?

If so, it contradicts my local observation.

```
# I confirmed vtorc failed to run PRS. The current main branch's behaivior.
(base) examples/compose❯ docker-compose logs -f vtorc
...
vtorc_1                           | 2022-07-02 04:23:31  WARNING orchestrator.go:250]  DiscoverInstance(vttablet101:3306) instance is nil in 0.004s (Backend: 0.002s, Instance: 0.003s), error=durability policy  not found
...
```

```
# After that, I ran SKDP manually (meaning over 15 seconds after vtorc startup).
(base) examples/compose❯ docker-compose exec vttablet101 /bin/bash
vitess@69cfce7aa851:/$ date
Sat Jul  2 04:27:31 UTC 2022
vitess@69cfce7aa851:/$ /vt/bin/vtctldclient --server vtctld:15999 SetKeyspaceDurabilityPolicy --durability-policy=none test_keyspace
ERROR: logging before flag.Parse: E0702 04:27:56.655676     724 syslogger.go:149] can't connect to syslog
{
  "keyspace": {
    "served_froms": [],
    "keyspace_type": 0,
    "base_keyspace": "",
    "snapshot_time": null,
    "durability_policy": "none"
  } 
} 

# I waited for something happened, but nothing.
vitess@69cfce7aa851:/$ date
Sat Jul  2 04:46:04 UTC 2022

# I ran PRS manually.
vitess@69cfce7aa851:/$ ./vt/bin/vtctlclient -server vtctld:15999 PlannedReparentShard --keyspace_shard='test_keyspace/-80' --new_primary=test-101
W0702 04:49:02.704957    1164 flag.go:109] Use of single-dash long flags is deprecated and will be removed in the next version of Vitess. Please use --server instead
W0702 04:49:02.706257    1164 flag.go:123] Detected a dashed argument after a positional argument. Currently these are treated as posargs that may be parsed by a subcommand, but in the next version of Vitess they will be parsed as top-level flags, which may not be defined, causing errors. To preserve existing behavior, please update your invocation to include a "--" after all top-level flags to continue treating --keyspace_shard=test_keyspace/-80 as a positional argument.
W0702 04:49:02.706498    1164 flag.go:123] Detected a dashed argument after a positional argument. Currently these are treated as posargs that may be parsed by a subcommand, but in the next version of Vitess they will be parsed as top-level flags, which may not be defined, causing errors. To preserve existing behavior, please update your invocation to include a "--" after all top-level flags to continue treating --new_primary=test-101 as a positional argument.

# test-101 turned healthy at the time I ran PRS (04:49:02) not around SKDP + ~ 15 seconds (04:27:56 ~ 04:28:11).
vitess@69cfce7aa851:/$ curl -s http://localhost:8080/debug/status
...
<tr class="healthy">
    <td>Jul 2, 2022 at 04:49:02 (UTC)</td>
    <td>healthy</td>
    <td>primary</td>
  </tr>

  <tr class="unhealthy">
    <td>Jul 2, 2022 at 04:23:33 (UTC)</td>
    <td>not serving: no replication status (errno 100) (sqlstate HY000)</td>
    <td>replica</td>
  </tr>
...
```

Reply:
> It will just take some time because the keyspace and shard informations are reloaded every 15 seconds, so vtorc will run PRS, but after 15 seconds.

Or it means that vtorc reloads the keyspace and shard information every 15 seconds,
but not try to run PRS every 15 seconds? It makes sense to me 😸

Reply:
Yep, you figured it out, it is the last one. It reloads the keyspace and shard information every 15 seconds. Once it does, then the discovery of vttablets will also work fine since it will have the durability policy and then PRS will also work.